radixdlt-core/radixdlt/src/main/java/com/radixdlt/client/handler/HighLevelApiHandler.java

```diff
@@ -213,6 +224,13 @@ private JSONObject formatTokenBalances(RadixAddress radixAddress, List<TokenBala
 			.put("tokenBalances", fromList(balances, TokenBalance::asJson));
 	}
 
+	private JSONObject formatStakePositions(List<BalanceEntry> balances) {
+		var array = fromList(balances, balance ->
+			jsonObject().put("validator", balance.getDelegate()).put("amount", balance.getAmount())
```

Comment:
does this really sum them up? looks like replace?

Reply:
No, it does not aggregate anything because this happens at the DB layer.